docs/modders/Bonus/Bonus_Types.md

```diff
@@ -1023,3 +1023,7 @@ Internal bonus, do not use
 ### DISINTEGRATE
 
 After death of unit no corpse remains
```

Comment:
Just noticed this - please add clarification that bonus is checked on dying unit, not on attacker unit - this is not "unit has disintegrating attack", but "unit body disintegrates on death"
```suggestion
When a unit affected by this bonus dies, no corpse is left behind.
```